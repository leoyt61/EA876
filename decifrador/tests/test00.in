HEEHBWRU THR XRY USPJIZUAHB DWGKWVTR IFAE GQUMZO!
