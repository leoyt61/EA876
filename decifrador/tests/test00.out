PARABENS POR TER CONSEGUIDO DECIFRAR ESTE CODIGO!
